supervision/dataset/core.py

```diff
@@ -46,6 +46,14 @@ def split(
     ) -> Tuple[BaseDataset, BaseDataset]:
         pass
 
+    def add_instance(
```

Comment:
@hardikdava looks to me like this change is not used.

Reply:
Sorry, it was a mixed up in commits. But it is not needed so I removed it.